src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -327,8 +351,15 @@ blockToMarkdown' opts (Div attrs ils) = do
   contents <- blockListToMarkdown opts ils
   variant <- asks envVariant
   return $
-    case () of
-         _ | isEnabled Ext_fenced_divs opts &&
+     case () of
+         _ | variant == Markua -> 
+                   case () of
+                        () | classes' == ["blurb"] -> prefixed "B> " contents <> blankline
+                           | classes' == ["aside"] -> prefixed "A> " contents <> blankline
```

Comment:
Blurbs can have other classes attached (e.g. in the documentation, `warning`).
So we shouldn't assume that there will be just one class, `blurb`. Instead, test to see that `"blurb"` is an element of `classes'`, and ditto for aside.

Reply:
we changed it to handle other classes aswell